### ISSUE TITLE
InputList missing options (waiting on QA to add vocab)

### ISSUE THREAD

**Issue by justinlittman** [LD4P/sinopia_editor#2587]
Steps to reproduce:
1. Open https://stage.sinopia.io/editor/ld4p:RT:bf2:AdminMetadata.
2. Under Assigning Agency, click "Choose..."

Expected result:
A list of organizations.

Actual result:
Nothing.

**Comment by justinlittman** [comment]
Turns out that authority is now only available as a bulk download: https://id.loc.gov/vocabulary/organizations.html

This will need to be turned into a QA authority.

cc @michelleif @jermnelson

**Comment by jgreben** [comment]
I submitted an QA indexing issue: https://github.com/LD4P/qa_server/issues/376

**Comment by justinlittman** [comment]
Closing in favor of QA ticket.
